xonsh/ptk_shell/formatter.py

```diff
@@ -0,0 +1,68 @@
+import builtins
```

Comment:
Please add docstrings to this module as well as documentation hooks in the `docs/` dir

Reply:
done.